README.md

```diff
@@ -311,7 +309,6 @@ described in [RFC 8032]
  * [link-header](https://github.com/spider-gazelle/link-header) - HTTP Link Header Parser
  * [ntlm](https://github.com/spider-gazelle/ntlm) - NTLM authentication
  * [proxy-fetcher.cr](https://github.com/nbulaj/proxy-fetcher.cr) - Proxy lists fetching & validating library
- * [sse.cr](https://github.com/y2k2mt/sse.cr) - [Server-Sent Events](https://www.w3.org/TR/2009/WD-eventsource-20090421) client
```

Comment:
This one should just have it's W3 link updated as the GH shard is still around it seems.